api/src/resources/project.rs

```diff
@@ -27,8 +27,17 @@ impl FromStr for ProjectName {
     }
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq, Hash)]
+pub struct Id(pub String);
+
 #[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
 pub struct Project {
+    // CODE HEALTH: At time of addition, deployed versions of reinfer may not
+    // return project ids.
+    //
+    // This can be removed once reinfer is deployed everywhere.
+    #[serde(default)]
+    pub id: Option<Id>,
```

Comment:
Do you know if we can remove the option now?

Reply:
I doubt it - I don't think we've deployed at UBS in the last 11 days.

Reply:
Ok, cool, I'll merge this if you have no objections